src/oph/heratepalvelu/amis/AMISherateHandler.clj

```diff
@@ -52,7 +52,8 @@
                   (:sisältyyOpiskeluoikeuteen opiskeluoikeus) ":" summary)
 
         :else (ac/check-and-save-herate! herate opiskeluoikeus koulutustoimija
-                                         (:ehoks herate-sources))))
+                                         (:ehoks herate-sources)))
+      (ac/update-herate-ehoks! (:ehoks-id herate) (:kyselytyyppi herate)))
```

Comment:
Pitäiskö tää olla
```clojure
:else (do
        (ac/check-and-save-herate! herate opiskeluoikeus koulutustoimija
                                   (:ehoks herate-sources))
        (ac/update-herate-ehoks! (:ehoks-id herate) (:kyselytyyppi herate))))
```
?

Reply:
Mun ajatuksena oli että tuota `ac/update-herate-ehoks!` kutsuttaisiin myös niissä tapauksissa, kun herätettä ei tallenneta Herätepalveluun, jotta kyseinen heräte ei tule enää uudelleen vastaavaan käsittelyyn. Eli sen takia siirsin kutsun tuohon `cond`in ulkopuolelle. Tep-puolen jaksoHandlerissa olikin jo vastaava menettely.